sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -113,7 +113,7 @@
     "@opentelemetry/api": "^1.0.1",
     "@opentelemetry/core": "^0.22.0",
     "@opentelemetry/resources": "^0.22.0",
-    "@opentelemetry/semantic-conventions": "^0.22.0",
+    "@opentelemetry/semantic-conventions": "^0.24.0",
```

Comment:
> The shrinkwrap file contains the following issues:
  Missing dependency "@opentelemetry/semantic-conventions" (^0.24.0) required by "@azure/monitor-opentelemetry-exporter"

The build is failing due to a missing update to the lock file. Can you run `rush update --recheck` and include the changes from that? Due to this build failure no tests are run either